src/test/fixtures/composes.ts

```diff
@@ -275,8 +280,49 @@ export const mockComposes: ComposesResponseItem[] = [
       ],
     },
   },
+  {
+    id: '1679d95b-8f1d-4982-8c53-8c2afa4ab04c',
+    image_name: 'testOscapImageName',
+    created_at: '2021-04-27T12:31:12Z',
+    request: {
+      distribution: RHEL_8,
+      image_requests: [
+        {
+          architecture: 'x86_64',
+          image_type: 'aws',
+          upload_request: {
+            type: 'aws',
+            options: {
+              share_with_accounts: ['123123123123'],
+            },
+          },
+        },
```

Comment:
As we need this compose for testing the Recreate functionality it might be nice to change it to any of the aws.s3 targets so we get the Recreate image instead of Launch for easier manual testing.

![Screenshot from 2023-10-05 12-31-01](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/b49b1537-d6bb-4286-8b96-a26b6a65567d)


But this also feels like a very opinionated nit pick so feel free to ignore it :sweat_smile: